CMakeLists.txt

```diff
@@ -6,6 +6,11 @@ set(ROS_VERSION "2")
 set(ROS_DISTRO "bouncy")
 set(ROS_PYTHON_VERSION "3")
 
+# allow overriding the distro name
+if(NOT $ENV{ROS_DISTRO_OVERRIDE} STREQUAL "")
```

Comment:
This line results in a warning when CMake is invoked with `--warn-uninitialized`. See https://github.com/ros2/rosidl_typesupport_connext/blob/e9d8da960abc691b7f304fabd256d97d006281ee/rosidl_typesupport_connext_cpp/CMakeLists.txt#L5-L6 for an example how to check if the environment variable is defined.

Reply:
What is the policy you want to apply to this repository ?

Comply with more rigid CMake flags only on the bouncy branch and onward? or update everywhere?

Reply:
I don't mind if it is backported. As long as the lastest development branch has the fix.

Reply:
Fixed

Reply:
Thanks.